changes/03e912acf20e02eb0324b4427f3dd596.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: The inet:rc2822:addr type now rejects malformed inputs which could cause incorrect
```

Comment:
```suggestion
desc: The ``inet:rc2822:addr`` type now rejects malformed inputs which could cause incorrect
```

Reply:
Also has a typo of `rc2822` instead of `rfc2822`